frequency: 10
frame_id: "imu_link"

# Hardware parameters
i2c_bus: 1
device_addr: 0x68
